src/views/List.jsx

```diff
@@ -20,52 +20,58 @@ export function List({ data, listPath }) {
 	});
 
 	function handleInputChange(e) {
+		e.preventDefault();
 		setInput(e.target.value);
 	}
 
 	function clearSearch() {
 		setInput('');
 	}
 
+	const listName = listPath?.substring(listPath.indexOf('/') + 1);
+
+	if (data.data.length === 0) {
+		return (
+			<>
+				<h2>{listName}</h2>
+				<AddItemForm listPath={listPath} />
+			</>
+		);
+	}
+
 	if (data.loading) {
 		// If data is not loaded yet, render a loading indicator
 		return <p>Loading...</p>;
 	}
 
 	return (
 		<>
-			<form action="">
-				<label htmlFor="searchItems">Search Items: </label>
-				<input
-					onChange={handleInputChange}
-					id="searchItems"
-					value={input}
-					type="text"
-				/>
-			</form>
-			<button onClick={clearSearch}>X</button>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<h2>{listName}</h2>
+			<div className="searchInput">
+				<form action="" onSubmit={(e) => e.preventDefault()}>
+					<label htmlFor="searchItems">Search your shopping list: </label>
+					<input
+						onChange={handleInputChange}
+						id="searchItems"
+						value={input}
+						type="text"
+					/>
+				</form>
+				<button onClick={clearSearch}>Clear</button>
+			</div>
+			<br />
```

Comment:
same note on this br, definitely avoid using these in a real app, always use padding/margin to achieve spacing. Also wrapping something in a div applies a `display: block` for you so it would this just wrapping this content in a `<div>`